odl/solvers/advanced/proximal_operators.py

```diff
@@ -369,15 +730,57 @@ def _call(self, x, out):
                 out.ufunc.maximum(lam, out=out)
 
                 # Global scaling
-                out /= lam
+                out /= (lam)
 
                 # Pointwise division
                 out.divide(diff, out)
 
     return _ProximalConvConjL1
 
 
-# TODO: move notes to ODL doc
+def proximal_l1(space, lam=1, g=None):
+    """Proximal operator factory of the l1-semi-norm.
+
+    Function for the proximal operator of the functional F where F is an
+    l1-semi-norm
+
+        F(x) = lam || ||x-g||_p ||_1
+
+    with x and g elements in ``space``, scaling factor lam, and point-wise
+    magnitude ||x||_p of x. If x is vector-valued, ||x||_p is the point-wise
```

Comment:
`pointwise_norm` is an OK name, but im not sure if it expresses the intent. You need to be aware that this will be used EVERY TIME someone wants the isotropic TV norm.

Perhaps name it `isotropic`? Obviously not optimal, but gives the correct "feel" when users want to use the isotropic TV-norm.

Regarding `PointwiseNorm` this is not the full story. For example, I'm not sure how the proximal is affected by for example a weighted norm.